packages/toolkit/src/query/react/buildHooks.ts

```diff
@@ -55,8 +55,8 @@ import { useShallowStableValue } from './useShallowStableValue'
 // Copy-pasted from React-Redux
 export const useIsomorphicLayoutEffect =
   typeof window !== 'undefined' &&
-  typeof window.document !== 'undefined' &&
-  typeof window.document.createElement !== 'undefined'
+  window.document &&
+  window.document.createElement
     ? useLayoutEffect
     : useEffect
```

Comment:
Can this be even shorter? It looks like there are [many implementations](https://github.com/Andarist/use-isomorphic-layout-effect/blob/master/src/index.ts) that are just:

`typeof document !== 'undefined' ? useLayoutEffect : useEffect`

Reply:
Hmm. I would normally say "those other implementations are probably wrong", but Andarist normally knows what he's doing.

I'm going to say leave this alone - it's there, we know it works.